### PR TITLE
BVPS-182 Parsing SFTP Data Files

### PR DESCRIPTION
**Pull request by emilymendelson**

### CODE REVIEW

File: Dockerfile
Comment (anayanapalli) [inline Dockerfile:1]: @emilymendelson  Can you add

`$ pip install -r requirements.txt` instead of `paramiko`?

**Review by anayanapalli — COMMENTED** [review]
Added a comment to fix the Dockerfile.

File: Dockerfile
Comment (emilymendelson) [reply Dockerfile:1]: Fixed!

**Review by anayanapalli — APPROVED** [review]
LGTM.

**Pull request merged**